### DISCUSSION TITLE
Voltage adjustment frequency

### DISCUSSION THREAD

**Discussion by User-462** [srozum/film_camera_tester#50] [Q&A] [answered]
How frequently do you calibrate the unit? I'm wondering if the voltage adjustment and measurement would be better on the outside of the unit to avoid disassembling the case each time.

**Comment by srozum** [comment, marked as answer]
Once. Had no need to touch it in two years.

**Comment by ic-racer** [comment]
I have not needed to re-adjust mine in 6 months. There is plenty of room in the housing, a digital volt readout could easily be added to the housing. Or even just two test points.
